README.md

```diff
@@ -24,12 +77,12 @@ docker compose -f docker-compose.prometheus-example.yaml up
 And then explore the generated links by opening the [main
 file](./examples/web/cmd/main.go) in your editor.
 
-## Quickstart
+# Quickstart
 
 There is a one-time setup phase to prime the code for autometrics. Once this
 phase is accomplished, only calling `go generate` is necessary.
 
-### Install the go generator.
+#### 1. Install the go generator.
```

Comment:
i wouldnt go from `#` to `####`, maybe do like `#` to `###` (it looks weird in the toc generated by gh, see comment above)